sshd/zenith/sshd/tunnel.py

```diff
@@ -206,17 +214,15 @@ def consul_check_service_host_and_port(server_config, tunnel):
     )
     response = requests.get(url, params = params)
     if 300 <= response.status_code < 200:
-        print("[SERVER] [ERROR] Failed to list existing services", file = sys.stderr)
-        sys.exit(1)
+        logger.critical("Failed to list Consul services")
     # The response should be empty, otherwise the tunnel is not allowed
     if response.json():
-        print("[SERVER] [ERROR] Service already exists for specified port", file = sys.stderr)
-        sys.exit(1)
+        logger.critical("Consul service already exists for specified port")
```

Comment:
Don't we still need to raise an exception here? I think we should log the params here too?

Reply:
And again

Reply:
We could log the params.